src/new-components/heroes/heroes.stories.mdx

```diff
@@ -0,0 +1,59 @@
+import { Canvas, Meta } from '@storybook/addon-docs';
+import { AuroraHero } from './aurora-hero'; import { AuroraType } from '../../components/aurora/aurora';
+
+<Meta
+  title="Layout/Heroes" />
+
+# Aurora Hero
+
+<Canvas withSource="open">
+  <AuroraHero title="Satellytes" kicker="Great stuff here">
+    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Accusantium animi, 
+    culpa deserunt enim explicabo laborum minima, nesciunt nisi nobis non odio 
+    quaerat quia quibusdam quisquam quos rerum sunt tempore unde!
+  </AuroraHero>
+</Canvas>
+
+## Different Aurora type
+<Canvas withSource="open">
+  <AuroraHero title="Satellytes" auroraType={AuroraType.Pink}>
+    Welcome folks!
+  </AuroraHero>
+</Canvas>
+
+# Image Hero
+Due to the gatsby image dependency no preview is available.
```

Comment:
That is so sad. The storybook is half useful when we are not able to preview images that rely on gatsby image 😞